packages/amplify-migration-e2e/src/gen1ResourceDetailsFetcher.ts

```diff
@@ -0,0 +1,42 @@
+import path from 'path';
+import { RefactorCategory } from './templategen';
+import { getProjectMeta } from '@aws-amplify/amplify-e2e-core';
+import { assertIdentityPool, assertStorage, assertUserPool, assertUserPoolClients } from './assertions';
+import { getResourceDetails } from './sdk_calls';
+
+async function getGen1AuthResourceDetails(projRoot: string) {
+  const gen1ProjRoot = path.join(projRoot, '.amplify', 'migration');
+  const gen1Meta = getProjectMeta(gen1ProjRoot);
+  const gen1Region = gen1Meta.providers.awscloudformation.Region;
+  const { gen1UserPoolId } = await assertUserPool(gen1Meta, gen1Region);
+  const { gen1IdentityPoolId } = await assertIdentityPool(gen1Meta, gen1Region);
+  const { gen1ClientIds } = await assertUserPoolClients(gen1Meta, gen1Region);
+  const [gen1ClientIdWeb, gen1ClientId] = gen1ClientIds;
```

Comment:
This variable will be used later, there is a bug which does creates only one UserPoolClient instead of 2. Once, that bug is resolved, the second UserPoolClient will also be used for assertions.